docs/resources/lb_pool_v2.md

```diff
@@ -28,63 +28,71 @@ resource "flexibleengine_lb_pool_v2" "pool_1" {
 
-    + When the protocol used by the listener is UDP, the protocol of the backend pool must be UDP.
-    + When the protocol used by the listener is TCP, the protocol of the backend pool must be TCP.
-    + When the protocol used by the listener is HTTP or TERMINATED_HTTPS, the protocol of the backend pool must be HTTP.
+* `name` - (Optional, String) Human-readable name for the pool.
 
-    Changing this creates a new pool.
+* `description` - (Optional, String) Human-readable description for the pool.
 
 * `loadbalancer_id` - (Optional, String, ForceNew) The load balancer on which to provision this
-    pool. Changing this creates a new pool.
-    Note: One of LoadbalancerID or ListenerID must be provided.
+  pool. Changing this creates a new pool.
+  Note: One of LoadbalancerID or ListenerID must be provided.
 
 * `listener_id` - (Optional, String, ForceNew) The Listener on which the members of the pool
-    will be associated with. Changing this creates a new pool.
-    Note: One of LoadbalancerID or ListenerID must be provided.
-
-* `lb_method` - (Required, String) The load balancing algorithm to
-    distribute traffic to the pool's members. Must be one of
-    ROUND_ROBIN, LEAST_CONNECTIONS, or SOURCE_IP.
+  will be associated with. Changing this creates a new pool.
+  Note: One of LoadbalancerID or ListenerID must be provided.
 
-* `persistence` - (Optional, List, ForceNew) Omit this field to prevent session persistence.  Indicates
-    whether connections in the same session will be processed by the same Pool
-    member or not. Changing this creates a new pool.
+* `persistence` - (Optional, List, ForceNew) Omit this field to prevent session persistence. Indicates
+  whether connections in the same session will be processed by the same Pool member or not.
+  The [persistence](#lb_persistence) object structure is documented below.
+  Changing this creates a new pool.
 
-* `admin_state_up` - (Optional, Bool) The administrative state of the pool.
-    A valid value is true (UP) or false (DOWN).
-
-The `persistence` argument supports:
+<a name="lb_persistence"></a>
+The `persistence` block supports:
 
 * `type` - (Required, String, ForceNew) The type of persistence mode. The current specification
-    supports SOURCE_IP, HTTP_COOKIE, and APP_COOKIE.
+  supports SOURCE_IP, HTTP_COOKIE, and APP_COOKIE.
 
 * `cookie_name` - (Optional, String, ForceNew) The name of the cookie if persistence mode is set
-    appropriately. Required if `type = APP_COOKIE`.
+  appropriately. It is Required if `type = APP_COOKIE`.
 
 * `timeout` - (Optional, Int, ForceNew) Specifies the sticky session timeout duration in minutes. This parameter is
   invalid when type is set to APP_COOKIE. The value range varies depending on the protocol of the backend server group:
 
   + When the protocol of the backend server group is TCP or UDP, the value ranges from 1 to 60.
   + When the protocol of the backend server group is HTTP or HTTPS, the value ranges from 1 to 1440.
```

Comment:
```suggestion
  + When the protocol of the backend server group is HTTP or HTTPS, the value ranges from 1 to 1440.
  
  Changing this will create a new resource.
```